capture/moloch.h

```diff
@@ -571,6 +572,8 @@ typedef struct molochpacket_t
     uint8_t        ipProtocol;     // ip protocol
     uint8_t        mProtocol;      // moloch protocol
     uint8_t        readerPos;      // position for filename/ops
+    uint8_t        ethCount;       // the number of ethernet frames found
+    uint32_t       ethOffset[MOLOCH_ETHER_MAX_STORE];  // offset to last ethernet frame from start
```

Comment:
At most this should be a uint16_t, although i wonder if it could be uint8_t

Reply:
it's about +50 per new mac in what I have seen. so when we do 7 we need to store a value like 350 or more. I will change this to uint16_t

Reply:
+50 bytes further in the packet data pointer.
I needed to store all the offsets like this because I  didnt see another way to reach the session pointer.
If I understand how to reparse instead of storing all mac offsets it in the packet pointer, I can redo this later.
First I will commit all requested changes